OpenRA.Mods.Common/Widgets/SupportPowerTimerWidget.cs

```diff
@@ -19,47 +19,165 @@ namespace OpenRA.Mods.Common.Widgets
 {
 	public class SupportPowerTimerWidget : Widget
 	{
+		public readonly int XObserver = 10;
+		public readonly int YObserver = 10;
+		public enum TimerOrder { AlliesFirst, EnemiesFirst }
+		public readonly TimerOrder Order = TimerOrder.EnemiesFirst;
+		[Translate] public readonly string HeaderAllies = "Allies";
+		[Translate] public readonly string HeaderEnemies = "Enemies";
 		public readonly string Font = "Bold";
 		public readonly string Format = "{0}: {1}";
-		public readonly TimerOrder Order = TimerOrder.Descending;
 
-		readonly IEnumerable<SupportPowerInstance> powers;
-		Pair<string, Color>[] texts;
+		readonly IEnumerable<SupportPowerInstance> allPowers;
+		Player[] allPlayers, playersSelf, playersAllies, playersEnemies;
+		Pair<string, Color>[] allTexts, textsSelf, textsAllies, textsEnemies;
+		delegate void Del();
```

Comment:
How else can I define a method reference with no return and no parameters? I tried doing it with Action but that requires System namespace which is not included here. Should I add a using System line and use Action?